Improve documentation on configuring plugin (#112)
Automattic/wp-openid-connect-server: README.md

openssl rsa -in oidc.key -pubout -out public.key
~~~

> And make them available to the plugin as below (this needs to be added before WordPress loads). This should be added to the `wp-config.php` file of your WordPress installation. Note, that it is important to add the `define` statements before the line `require_once ABSPATH . 'wp-settings.php';`. Otherwise, your RSA keys might not be visible to the rest of the WordPress website.

Reviewer: While that's one way to do it. We purposefully don't recommend a specific way.

> And make them available to the plugin as below (this needs to be added before WordPress loads). This should be added to the `wp-config.php` file of your WordPress installation. Note, that it is important to add the `define` statements before the line `require_once ABSPATH . 'wp-settings.php';`. Otherwise, your RSA keys might not be visible to the rest of the WordPress website.

Reviewer: ```suggestion
And make them available to the plugin as follows (this needs to be added before WordPress loads):
```

### Define the clients

> Define your clients by adding a filter to `oidc_registered_clients` in a separate plugin file or `functions.php` of your theme or in a MU-plugin as below. The easiest way would be to add filter through WordPress Admin interface by going to Appearance -> Theme file editor -> choose Theme Functions (functions.php) on the right hand side menu. You can add the code to the end of the file before `?>`.

Reviewer: While that's one way to do it. We purposefully don't recommend a specific way.

> Define your clients by adding a filter to `oidc_registered_clients` in a separate plugin file or `functions.php` of your theme or in a MU-plugin as below. The easiest way would be to add filter through WordPress Admin interface by going to Appearance -> Theme file editor -> choose Theme Functions (functions.php) on the right hand side menu. You can add the code to the end of the file before `?>`.

Reviewer: ```suggestion
Define your clients by adding a filter to `oidc_registered_clients` in a separate plugin file or `functions.php` of your theme or in a MU-plugin like:`.
```

}
~~~

> ### Check that everything works as expected

Reviewer: @MariaMozgunova Thanks for the contributions! Happy to include the change here without the changes in other 2 sections.
